modules/core/src/main/scala/org/scalasteward/core/vcs/VCSApiAlg.scala

```diff
@@ -23,24 +23,24 @@ import org.scalasteward.core.git.Branch
 import org.scalasteward.core.vcs.data._
 import org.scalasteward.core.util.MonadThrowable
 
-trait GitHubApiAlg[F[_]] {
+trait VCSApiAlg[F[_]] {
 
-  /** https://developer.github.com/v3/repos/forks/#create-a-fork */
-  def createFork(repo: Repo): F[RepoOut]
+  def createFork(repo: Repo)(implicit F: MonadThrowable[F]): F[RepoOut]
```

Comment:
Since this is a requirement of one of the implementing classes, I'd prefer to have it as implicit parameter on the  class, e.g. `class Http4sGitLabApiAlg[F[_]: MonadThrowable]`.

Reply:
Done 👍